portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUI.testcase

```diff
@@ -74,4 +141,140 @@ definition {
 			vocabularyName = "Vocabulary Name");
 	}
 
+	@description = "This is a use case for LPS-107730. Can input 254 characters to name field, key field and value field of category."
+	@priority = "4"
+	test Input254CharactersIntoNameFieldKeyFieldAndValueField {
+		var inputText = "The number of characters is 254.The number of characters is 254.The number of characters is 254.The number of characters is 254.The number of characters is 254.The number of characters is 254.The number of characters is 254.The number of characters is 25";
+
+		task ("Assert input 254 characters to name field of category") {
+			Category.openCategoriesAdmin(siteURLKey = "test-site-name");
+
+			Vocabulary.addCP(vocabularyName = "Vocabulary Name");
+
+			LexiconEntry.gotoEntryMenuItem(
+				menuItem = "Add Category",
+				rowEntry = "Vocabulary Name");
+
+			PortletEntry.inputTitle(title = "${inputText}");
+
+			AssertTextEquals(
+				locator1 = "TextInput#TITLE",
+				value1 = "${inputText}");
+
+			PortletEntry.save();
+
+			AssertVisible(
+				key_tableEntry = "${inputText}",
+				locator1 = "LexiconTable#TABLE_ENTRY");
+		}
+
+		task ("Assert input 254 characters to key field and value field of category") {
+			LexiconEntry.gotoEntryMenuItem(
+				menuItem = "Edit",
+				rowEntry = "${inputText}");
+
+			Navigator.gotoNavUnderline(navUnderline = "Properties");
+
+			Type(
+				locator1 = "CategoriesEditCategory#PROPERTIES_KEY_FIELD_1",
+				value1 = "${inputText}");
+
+			AssertTextEquals(
+				locator1 = "CategoriesEditCategory#PROPERTIES_KEY_FIELD_1",
+				value1 = "${inputText}");
+
+			Type(
+				locator1 = "CategoriesEditCategory#PROPERTIES_VALUE_FIELD_1",
+				value1 = "${inputText}");
+
+			AssertTextEquals(
+				locator1 = "CategoriesEditCategory#PROPERTIES_VALUE_FIELD_1",
+				value1 = "${inputText}");
+
+			PortletEntry.save();
+
+			LexiconEntry.gotoEntryMenuItem(
+				menuItem = "Edit",
+				rowEntry = "${inputText}");
+
+			Navigator.gotoNavUnderline(navUnderline = "Properties");
+
+			AssertTextEquals(
+				locator1 = "CategoriesEditCategory#PROPERTIES_KEY_FIELD_1",
+				value1 = "${inputText}");
+
+			AssertTextEquals(
+				locator1 = "CategoriesEditCategory#PROPERTIES_VALUE_FIELD_1",
+				value1 = "${inputText}");
+		}
+	}
+
+	@description = "This is a use case for LPS-107730. Can input 255 characters to name field, key field and value field of category."
+	@priority = "5"
+	test Input255CharactersIntoNameFieldKeyFieldAndValueField {
+		var inputText = "The number of characters is 255.The number of characters is 255.The number of characters is 255.The number of characters is 255.The number of characters is 255.The number of characters is 255.The number of characters is 255.The number of characters is 255";
```

Comment:
It has 256 characters, is it right?

Reply:
The value of inputText has 255 characters.